ngraph/test/files/paddlepaddle/gen_scripts/generate_rnn_lstm.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from save_model import saveModel
+import sys
+
+
+def pdpd_rnn_lstm(input_size, hidden_size, layers, direction):
+    import paddle as pdpd
+    pdpd.enable_static()
+    main_program = pdpd.static.Program()
+    startup_program = pdpd.static.Program()
+    w = np.ones((4  * hidden_size, input_size)).astype(np.float32)
+    r = np.ones((4 * hidden_size, hidden_size)).astype(np.float32)
```

Comment:
Hi @nosovmik 4 is the number of gates in LSTMCELL.  It cannot be 5/2. It's a fixed value in LSTMCell. And LSTM consists of multiple layers of LSTMCell.

https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/layer/rnn/LSTMCell_en.html

Reply:
Can we add 'assert' somewhere in c++ implementation? Otherwise, I can create LSTM layer in PaddlePaddle, it can infer layer. And I can convert it using Frontend, but inference result mismatches

Reply:
@nosovmik, the `w` and `r` are used to initialize the hidden states of LSTMCell. If we don't provide the needed shape `[4*hidden_size, input_size]`, the PDPD will use a default random initializer to init those hidden states and the sizes are fixed in their [code](https://github.com/PaddlePaddle/Paddle/blob/a3debea2bd79ea05219cfd69bc7b9865cac7b1b2/python/paddle/nn/layer/rnn.py#L484). I will check whether we can add a ASSERT in our conversion.

Reply:
@nosovmik I check the results return by PDPD and OV. If I change `4` to `5` or other values, the python script returns wrong value by PDPD framework. If we both load the saved  PDPD model from scratch, then no matter which value(4 or 5) is used, the results from PDPD and converted OV model are the same. It's highly that the PDPD framework doesn't handle the wrong hidden shape during the run-time graph construction. However the saved model contains the correct info. Therefore, I think our implementation is good for this part. And I ASSERT is not needed here.

Reply:
ok